analyses/cell-type-consensus/exploratory-notebooks/01-reference-exploration.Rmd

```diff
@@ -0,0 +1,622 @@
+```
+
+I think I'm in favor of not having a "blood cell" label, since I'm not sure that it's helpful. 
+Also, if two different methods label something a platelet and a neutrophil, then perhaps that label is inaccurate and it's really a tumor cell. 
+
+#### Bone cell
+
+```{r}
+# bone cell
+print_df |> 
+  dplyr::filter(cl_annotation == "bone cell")
+```
+
+I think I would also remove bone cell, since hematopoietic stem cells and osteoclasts seem pretty different to me. 
+
+#### Myeloid leukocyte
+
+```{r}
+# myeloid leukocyte cell
+print_df |> 
+  dplyr::filter(cl_annotation == "myeloid leukocyte")
+```
+
+I'm torn on this one, because I do think it's helpful to know if something is of the myeloid lineage, but if we aren't keeping lymphocyte then I would argue we shouldn't keep myeloid leukocyte. 
+Noting that after discussion we have decided to keep this one since T and B cells are much easier to differentiate based on gene expression alone than cells that are party of the myeloid lineage. 
+
+#### Progenitor cell
+
+```{r}
+# progenitor cell
+print_df |> 
+  dplyr::filter(cl_annotation == "progenitor cell") |> 
+  head(n=15) # there's a lot of these so let's only print out some 
+```
+
+Same with `progenitor cell`, I do think it could be helpful to know that something may be a progenitor cell, but when you have a cell with the label for HSC and the label for cells like monocytes or osteoblasts, then maybe we are talking about a tumor cell instead. 
+After discussion, we are going to remove progenitor cells.
+
+Along those same lines, I think the below terms, `lining cell` and `supporting cell`, are too broad even though they have few descendants. 
+
+#### Lining cell
+
+```{r}
+# lining cell 
+print_df |> 
+  dplyr::filter(cl_annotation == "lining cell")
+```
+
+#### Supporting cell
+
+```{r}
+# supporting cell 
+print_df |> 
+  dplyr::filter(cl_annotation == "supporting cell")
+```
+
+
+### Discarded cell types
+
+We can also look at what cell type labels we are excluding when using this cut off to see if there are any terms we might actually want to keep instead. 
+
+```{r}
+lca_df |> 
+  dplyr::filter(total_descendants > cutoff) |> 
+  dplyr::pull(cl_annotation) |> 
+  unique()
+```
+
+The only terms in this list that I would be concerned about losing are "neuron" and epithelial cells. 
+Let's look at those combinations. 
+
+#### Neuron
+
+```{r}
+# neuron
+print_df |> 
+  dplyr::filter(cl_annotation == "neuron")
+```
+
+It looks like there are a lot of types of neurons in the PanglaoDB reference and only "neuron" as a term in Blueprint. 
+Even though neuron has ~ 500 descendants, I think we should keep these labels. 
+
+#### Epithelial cell
+
+```{r}
+# epithelial cell
+print_df |> 
+  dplyr::filter(cl_annotation == "epithelial cell")
+```
+
+The PanglaoDB cell types seem to be more specific than the ones present in Blueprint Encode, similar to the observation with neurons.
+We should keep epithelial cell. 
```

Comment:
When we codify this, I would say epithelial cell is _only acceptable_ when the Blueprint annotation is `Epithelial cells` because it's likely just a matter of PanglaoDB being much more specific. When the Blueprint annotation is `Keratinocytes`, things seem to get a bit weird.